pallets/stateful-storage/src/types.rs

```diff
@@ -167,12 +167,20 @@ impl<PageDataSize: Get<u32>> Page<PageDataSize> {
 	}
 }
 
+/// PartialEq and Hash should be both derived or implemented manually based on clippy rules
 impl<PageDataSize: Get<u32>> Hash for Page<PageDataSize> {
 	fn hash<H: Hasher>(&self, state: &mut H) {
 		state.write(&self.data[..]);
 	}
 }
 
+/// PartialEq and Hash should be both derived or implemented manually based on clippy rules
+impl<PageDataSize: Get<u32>> PartialEq for Page<PageDataSize> {
```

Comment:
Could not `#derive(Hash)` since `BoundedVec` was not implemented the **Hash** so had to implement `PartialEq`